lib/streamlit/elements/widgets/button.py

```diff
@@ -673,15 +673,15 @@ def _download_button(
         element_id = compute_and_register_element_id(
             "download_button",
             user_key=key,
+            # download_button is allowed to be used in a form.
```

Comment:
nit: is not allowed to be used in a form

Reply:
Fixed 👍